pkg/network/master/master.go

```diff
@@ -52,7 +51,11 @@ func Start(networkConfig osconfigapi.MasterNetworkConfig, osClient *osclient.Cli
 	}
 
 	var err error
-	master.networkInfo, err = common.ParseNetworkInfo(networkConfig.ClusterNetworkCIDR, networkConfig.ServiceNetworkCIDR)
+	var clusterNetworkEntries []networkapi.ClusterNetworkEntry
+	for _, cidr := range networkConfig.ClusterNetworks {
+		clusterNetworkEntries = append(clusterNetworkEntries, networkapi.ClusterNetworkEntry{CIDR: cidr.CIDR, HostSubnetLength: cidr.HostSubnetLength})
+	}
+	master.networkInfo, err = common.ParseNetworkInfo(clusterNetworkEntries, networkConfig.ServiceNetworkCIDR)
```

Comment:
I use clusterNetworkEntries to convert between the ClusterNetworkEntry types. The one defined in the master (pkg/cmd/server/api/types.go) and the internal type for the sdn (pkg/network/apis/network/types.go)

Reply:
ClusterNetworkEntry struct in pkg/cmd/server/api/types.go and pkg/network/apis/network/types.go are exactly the same thing. Why not define once in pkg/cmd/server/api/types.go and reuse the same struct in pkg/network/apis/network/types.go ? Then you can directly pass networkConfig.ClusterNetworks to ParseNetworkInfo().

Reply:
@pravisankar I just tried to make the switch and reuse the code from pkg/cmd/server/api/types.go, it is illegal to import pkg/cmd/server/api/types.go into pkg/network/apis/network/types.go hack/verify-imports.sh complains

Reply:
@JacobTanenbaum 
I was surprised that we can't reuse the structs in types.go, so I looked at other types.go files.
Gist is that we can import/reuse type structs from other packages as long as there are no import cycles (A->B, B->A or A->B, B->C, C->A). Looks like you can reuse ClusterNetworkEntry network/types.go in api/types.go (just like resource.Quantity, runtime.Object, etc. in api/types.go).
Ideally we should duplicate ClusterNetworkEntry only when they really diverge in the future (and this may not happen). 
This is not critical and don't want to block on this, we can get this pr merged and you can submit a follow up pr.

Reply:
@pravisankar I will look at this tonight, if I can't get it in before this PR gets on the merge queue I will try to have this ready right after it lands

Reply:
@pravisankar so far only kuberenetes packages are imported into server/api/types.go, I don't know if it on purpose or coincidental so I think that discussion might be better left for a follow up PR